angular.json

```diff
@@ -40,8 +40,8 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "500mb",
-                  "maximumError": "1000mb"
+                  "maximumWarning": "2mb",
```

Comment:
I think this should still be included as the maximumWarning and maximumError were way too big to ever be impactful to ensure we are not including massive assets that we don't need to be.

Reply:
Can you make a ticket to review this configuration, then we can apply this changeset to a PR for that?  Sounds like we should probably review that configuration top to bottom in case there is other weird / outdated stuff to fix!

Reply:
I think this covers that: https://github.com/ua-snap/nccwsc-projects/issues/181